zerver/views/registration.py

```diff
@@ -73,6 +79,8 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
     key = request.POST['key']
     confirmation = Confirmation.objects.get(confirmation_key=key)
```

Comment:
Hmm, since `key` is a param input by the user to this endpoint, we'll get an exception here if someone POSTs garbage to it. Can you do a follow-up commit for `try/except Confirmation.DoesNotExist`` here to handle it more gracefully? Returning `confirmation_link_expired_error.html` page is probably the right thing to do in the failure case :thinking:

Reply:
Sure! It's done :smile: